### DISCUSSION TITLE
Weird/inconsistent behavior with dropdowns in editor when scrolling

### DISCUSSION THREAD

**Discussion by hwabis** [ppy/osu#18489] [Q&A] [answered]
Open a dropdown menu, then open a nested menu, and scroll in the first menu. Sometimes the second menu will follow the scroll, and other times it just gets stuck there.

https://user-images.githubusercontent.com/62194990/170891201-3fc68437-1727-4136-ae29-18bb9b322581.mp4

**Comment by smoogipoo** [comment, marked as answer]
Certainly looks like something regressed somewhere. There was another case that came up for me recently which I passed off as me doing something weird that I can't remember, something like scrolling the beatmap listing would also sometimes scroll song select

**Reply by smoogipoo** [reply]
https://github.com/ppy/osu/issues/18490
